### ISSUE TITLE
No session found

### ISSUE THREAD

**Issue by fbagjug** [SuperPykkon/minecraft-server-web-console#1]
Hello, When starting I get an error "Error, No screen session found." I dont understand why and would love if anyone knew the fix.

**Comment by SuperPykkon** [comment]
Hello there!
Firstly, this is a very old project I did for fun and it's full of serious security vulnerabilities. I'd recommend you to check out my other project GoMinewrap [here](https://github.com/SuperPykkon/GoMinewrap) which is pretty much a upgrade of this and it supports all platforms.

With that aside, the problem is you need to create a screen session first with `screen -S mcs` and you must have `screen` installed (if you don't). Also this only supports Linux and I don't know about Mac OS (I don't know if you can install screen there).

**Comment by fbagjug** [comment]
Yeah ill check that project out and I did use the "screen -S mcs" command but anyways.
